src/java/org/apache/cassandra/service/accord/AccordConfigurationService.java

```diff
@@ -420,14 +424,17 @@ void maybeReportMetadata(ClusterMetadata metadata)
     @Override
     protected void fetchTopologyInternal(long epoch)
     {
+        if (ClusterMetadata.current().epoch.getEpoch() < epoch)
```

Comment:
I think you need to rebase, Ariel added this and moved this function to run on `Stage.ACCORD_MIGRATION`

Reply:
Yep, I did that, rebased and pushed to `-rebased` branch while waiting for CI. Pushed now here too.